vehiclestorage/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
While I might have missed committing this file, it doesn't really fit with this change and should be done separately. Please remove it from this pull request.

Reply:
My IDE generated some files and since it belongs to Gradle, I tought it would be ok to publish them. No problem, I'll remove it. (Maybe another pull request to add them ?)

Reply:
I think there are more files that need to be added as well as I'm not using the same gradle version you are at the moment, but I need to review what all is intended to be committed for gradle and I'll follow up with a commit after with any changes that need to be done.